components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -76,4 +76,37 @@ class ActionSheetTest: XCTestCase {
     }
     
   }
+
+  func testDefaultBackgroundColor() {
+    // Then
+    XCTAssertEqual(actionSheet.backgroundColor, .white)
+    let subviewsArray = actionSheet.view.subviews
+    for view in subviewsArray {
+      XCTAssertEqual(view.backgroundColor, .white)
+    }
```

Comment:
Specifically we should test .view itself here because the default is nil.

Reply:
Done